### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by imeed166**
I removed `how to create db` section and changed `how to use` section because it gives people the wrong impression that you can use the plugin only with folders.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,13 +7,9 @@ This plugin is a Notion like database based on folders.
 - **[Documentation](https://rafaelgb.github.io/obsidian-db-folder/)**
 - **[Roadmap](https://github.com/RafaelGB/obsidian-db-folder/projects/1)**
 - **[changelog](https://rafaelgb.github.io/obsidian-db-folder/changelog/)**
-### How can I add a database?
-You can add a folder database by right clicking on the folder where you want to store your table
-
-<img src="https://github.com/RafaelGB/obsidian-db-folder/blob/master/docs/resources/AddDatabase.gif" width="200" height="250"/>
```
Comment (RafaelGB) [inline README.md:13]: Why did you remove all the media resources?

**Review by RafaelGB — CHANGES_REQUESTED** [review]
Use some media as support

File: README.md (hunk 1 above)
Comment (imeed166) [reply README.md:13]: Because It only shows the right-click folder method of creating the database. So the user will think that is the only way. I probably should've added more ways instead of removing it entirely, you're probably right.

Comment (RafaelGB) [reply README.md:13]: Ok, so new media could be added with future PRs

**Review by RafaelGB — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (imeed166) [reply README.md:13]: Yes, I was thinking when the 0.16 is out, I could add image/videos for README and for the docs.
